2264/3/helpers/comments_parser.rb

```diff
@@ -0,0 +1,35 @@
+require 'json'
+require 'mechanize'
+
+# Comments parser
+class CommentsParser
+  attr_reader :agent, :path_to_page
+
+  def initialize(link)
+    @agent = Mechanize.new
+    @path_to_page = link
+  end
+
+  def launch_parser
+    parse_comments_from_page
```

Comment:
method should call at least 2 other methods - otherwise U should remove it:
U can just put the body 'parse_comments_from_page' right here